start.py

```diff
@@ -569,10 +594,7 @@ def boot():
     # ------------------------------------------------------------------------
     # set OPENPYPE_ROOT to running location until proper version can be
     # determined.
-    if getattr(sys, 'frozen', False):
-        os.environ["OPENPYPE_ROOT"] = os.path.dirname(sys.executable)
-    else:
-        os.environ["OPENPYPE_ROOT"] = os.path.dirname(__file__)
+    os.environ["OPENPYPE_ROOT"] = BUILD_ROOT
```

Comment:
This seems a bit obsolete, why don't we set it directly to `OPENPYPE_ROOT` from the beginning?

Reply:
I agree that `BUILD_ROOT` is little confusing as that can point either to actually built (or frozen if you will) OpenPype root but also to live sources root. And when installed on user machine it will point to install location. `OPENPYPE_ROOT` makes more sense to me.

Reply:
I wanted use `OPENPYPE_ROOT` but current `OPENPYPE_ROOT` is used in different way and is already used at many places.

NOTE: `BUILD_ROOT` because I didn't know how else should I call it and didn't care much because it's a variable available only in start.py.